lib/cocoapods/command.rb

```diff
@@ -102,6 +104,14 @@ def ensure_master_spec_repo_exists!
 
     private
 
+    # Returns a new {Installer} parametrized from the {Config}.
+    #
+    # @return [Installer]
+    #
+    def installer_for_config
```

Comment:
I was trying to avoid calling it `installer` as disambiguation from local variable names. In advance I don't see any practical relevance for injecting a custom configuration.